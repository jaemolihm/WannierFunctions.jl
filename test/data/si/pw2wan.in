&inputpp
   outdir = 'temp/'
   prefix = 'si'
   seedname = 'si'
   spin_component = 'none'
   write_mmn = .true.
   write_amn = .true.
   write_dmn = .true.

!scdm_proj         = .true.
!scdm_entanglement = 'erfc'
!scdm_mu = 42.0
!scdm_sigma = 5.0
/
