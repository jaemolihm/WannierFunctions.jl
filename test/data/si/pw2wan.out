
     Program PW2WANNIER v.7.0 starts on 17Jul2022 at 18:16:48 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      20 processor cores
     Number of MPI processes:                20
     Threads/MPI process:                     1

     MPI processes distributed on     2 nodes
     R & G space division:  proc/nbgrp/npool/nimage =      20
     21310 MiB available memory on the printing compute node when the environment starts
 
 
  Reading nscf_save data

     Reading xml data from directory:

     temp/si.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PW
                           (   1   4   0   0   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          51      51     15                 1076     1076     186
     Max          52      52     16                 1079     1079     191
     Sum        1027    1027    313                21547    21547    3767
 
     Using Pencil Decomposition
 
     Reading collected, re-writing distributed wavefunctions
 
  Spin CASE ( default = unpolarized )
 
  Wannier mode is: standalone     
 
  -----------------
  *** Reading nnkp 
  -----------------
 
  Checking info from wannier.nnkp file
 
  - Real lattice is ok
  - Reciprocal lattice is ok
  - K-points are ok
  - Number of wannier functions is ok (  4)
  - All guiding functions are given 
 
 Projections:
    0.000000    0.000000    0.000000  0  1  1    3.000000
   -0.250000    0.000000    0.250000  0  1  1    3.000000
    0.000000    0.250000    0.250000  0  1  1    3.000000
   -0.250000    0.250000    0.000000  0  1  1    3.000000
 
  Reading data about k-point neighbours 
 
  All neighbours are found 
 
  Opening pp-files 
 
 
  ----------------
  *** Compute DMN 
  ----------------
 
  Number of symmetry operators =    48
      1-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.0000000      0.0000000
      2-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.2500000      0.0000000
      3-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.0000000      0.2500000
      4-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.2500000      0.2500000
      5-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.0000000      0.0000000
      6-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.2500000      0.0000000
      7-th symmetry operators is
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.2500000      0.2500000
      8-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.0000000      0.2500000
      9-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     10-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     11-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     12-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     13-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     14-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     15-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     16-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     17-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     18-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     19-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     20-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     21-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     22-th symmetry operators is
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     23-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     24-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     25-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.0000000      0.0000000
     26-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.2500000      0.0000000
     27-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.0000000      0.2500000
     28-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.2500000      0.2500000
     29-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      0.0000000      0.0000000
     30-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -0.2500000      0.2500000      0.0000000
     31-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      0.2500000      0.2500000
     32-th symmetry operators is
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -0.2500000      0.0000000      0.2500000
     33-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     34-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     35-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     36-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     37-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     38-th symmetry operators is
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     39-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     40-th symmetry operators is
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     41-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     42-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     43-th symmetry operators is
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     44-th symmetry operators is
      0.0000000      1.0000000      0.0000000
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      0.2500000      0.2500000
     45-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
     -0.2500000      0.2500000      0.0000000
     46-th symmetry operators is
      0.0000000      0.0000000     -1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
     -0.2500000      0.0000000      0.2500000
     47-th symmetry operators is
      0.0000000      0.0000000      1.0000000
      1.0000000      0.0000000      0.0000000
      0.0000000     -1.0000000      0.0000000
      0.0000000      0.0000000      0.0000000
     48-th symmetry operators is
      0.0000000      0.0000000      1.0000000
     -1.0000000      0.0000000      0.0000000
      0.0000000      1.0000000      0.0000000
      0.0000000      0.2500000      0.2500000


  DMN(d_matrix_wann): nir =       16
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16
  DMN(d_matrix_wann) calculated


  DMN(d_matrix_band): nir =       16
  DMN(d_matrix_band): nir in this pool =       16
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16
  DMN(d_matrix_band) calculated
 
  --------------------------
  *** Compute  A projections
  --------------------------
 
 Number of local k points =      216
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27      28      29      30
      31      32      33      34      35      36      37      38      39      40
      41      42      43      44      45      46      47      48      49      50
      51      52      53      54      55      56      57      58      59      60
      61      62      63      64      65      66      67      68      69      70
      71      72      73      74      75      76      77      78      79      80
      81      82      83      84      85      86      87      88      89      90
      91      92      93      94      95      96      97      98      99     100
     101     102     103     104     105     106     107     108     109     110
     111     112     113     114     115     116     117     118     119     120
     121     122     123     124     125     126     127     128     129     130
     131     132     133     134     135     136     137     138     139     140
     141     142     143     144     145     146     147     148     149     150
     151     152     153     154     155     156     157     158     159     160
     161     162     163     164     165     166     167     168     169     170
     171     172     173     174     175     176     177     178     179     180
     181     182     183     184     185     186     187     188     189     190
     191     192     193     194     195     196     197     198     199     200
     201     202     203     204     205     206     207     208     209     210
     211     212     213     214     215     216
  AMN calculated
 
  ---------------
  *** Compute  M 
  ---------------
 
 Number of local k points =      216
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14      15      16      17      18      19      20
      21      22      23      24      25      26      27      28      29      30
      31      32      33      34      35      36      37      38      39      40
      41      42      43      44      45      46      47      48      49      50
      51      52      53      54      55      56      57      58      59      60
      61      62      63      64      65      66      67      68      69      70
      71      72      73      74      75      76      77      78      79      80
      81      82      83      84      85      86      87      88      89      90
      91      92      93      94      95      96      97      98      99     100
     101     102     103     104     105     106     107     108     109     110
     111     112     113     114     115     116     117     118     119     120
     121     122     123     124     125     126     127     128     129     130
     131     132     133     134     135     136     137     138     139     140
     141     142     143     144     145     146     147     148     149     150
     151     152     153     154     155     156     157     158     159     160
     161     162     163     164     165     166     167     168     169     170
     171     172     173     174     175     176     177     178     179     180
     181     182     183     184     185     186     187     188     189     190
     191     192     193     194     195     196     197     198     199     200
     201     202     203     204     205     206     207     208     209     210
     211     212     213     214     215     216
  MMN calculated
 
  -----------------------------------
  *** Orbital terms are not computed 
  -----------------------------------
 
  -----------------------------------
  *** SHC terms are not computed 
  -----------------------------------
 
  ----------------
  *** Write bands 
  ----------------
 
 
  -----------------------------
  *** Plot info is not printed 
  -----------------------------
 
  -----------------------------
  *** Parity info is not printed 
  -----------------------------
 
  ------------
  *** Stop pp 
  ------------
 
 
     init_pw2wan  :      0.85s CPU      1.08s WALL (       1 calls)
     compute_dmn  :      6.66s CPU      6.80s WALL (       1 calls)
     compute_amn  :      4.88s CPU      4.91s WALL (       1 calls)
     compute_mmn  :      0.89s CPU      0.98s WALL (       1 calls)

     Internal routines:
     compute_u_kb :      0.79s CPU      0.82s WALL (    1728 calls)
 
     PW2WANNIER   :     13.32s CPU     13.81s WALL

 
   This run was terminated on:  18:17: 2  17Jul2022            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
 
